docs/docs/makefile.md

```diff
@@ -1,8 +1,10 @@
+## Windows
 In order to build an MTA archive from a project's source code in a Windows environment, you should install `GNU Make 4.2.1` on your machine.
 
-You can download the executable from a source that you trust according to your processor (32-bit or 64-bit) to a local folder (change the file name to `make.exe` )and then add the folder to the PATH environment variable.  For example, you can download the tool from [here](https://github.com/mbuilov/gnumake-windows).
-
+You can download the executable from a source that you trust according to your processor (32-bit or 64-bit) to a local folder (change the file name to `make.exe`) and then add the folder to the `PATH` environment variable. For example, you can download the tool from [here](https://github.com/mbuilov/gnumake-windows).
 
 Alternatively, you can use [Chocolatey](https://chocolatey.org/packages/make) to install or upgrade `GNU Make`.
 
 
+## Mac
+If the build fails because `make` cannot run you might have to install the Command-Line Tools. Please refer to the [troubleshooting](troubleshooting.md) topic for more details.
```

Comment:
If the build fails because `make` cannot run, you might have to install the Command-Line Tools. See the [troubleshooting](troubleshooting.md) topic for more details.

Reply:
Done